BeatSaberHTTPStatusPlugin.sln

```diff
@@ -1,6 +1,19 @@
+
 Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30011.22
+MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "BeatSaberHTTPStatus", "BeatSaberHTTPStatus\BeatSaberHTTPStatusPlugin.csproj", "{1C0571AC-F8D6-4F9E-93B2-2F23892AE61E}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution items", "Solution items", "{2623DD3B-3EC1-43FE-AD24-5D31D1E49F28}"
```

Comment:
Is the "Solution items" a special string or could it be renamed to something like "Useful files" or "LOOK AT THESE"?

Reply:
It's just a normal string so it's perfectly possible to rename it to something else.
I think you'll need the 2 strings in sync though.

Reply:
Renamed to "Useful files" as per conversation.